src/gui/editorwidgets/qgsvaluemapconfigdlg.h

```diff
@@ -36,8 +36,12 @@ class GUI_EXPORT QgsValueMapConfigDlg : public QgsEditorConfigWidget, private Ui
 
     void updateMap( const QMap<QString, QVariant> &map, bool insertNull );
 
+  private:
+    void setRow( int row, const QString value, const QString description );
+
   private slots:
     void vCellChanged( int row, int column );
+    void addNullButtonPushed();
```

Comment:
These slot names make me think of the [chemical brothers](https://www.youtube.com/watch?v=Xu3FTEmN-eg)